src/toolkits/coreml_export/neural_net_models_exporter.cpp

```diff
@@ -140,9 +141,9 @@ void set_image_feature_size_range(ImageFeatureType* image_feature,
 
 ImageFeatureType* set_image_feature(
     FeatureDescription* feature_desc, size_t image_width, size_t image_height,
-    std::string description = "",
+    std::string input_name, std::string description = "",
     ImageFeatureType::ColorSpace image_type = ImageFeatureType::RGB) {
-  feature_desc->set_name("image");
+  feature_desc->set_name(input_name);
```

Comment:
probably I miss some context here. is the input name passed by user side?

Reply:
the input_name will be inferred from the column of the feature column of the dataset.
It is not included in arguments of export_to_coreml() so the users don't pass it.